pkgs/test/test/runner/runner_test.dart

```diff
@@ -698,4 +703,108 @@ void main(List<String> args) async {
       await test.shouldExit(0);
     });
   });
+
+  group('nnbd', () {
+    final _testContents = '''
+import 'package:test/test.dart';
+import 'weak.dart';
+
+void main() {
+  test("success", () {
+    expect(foo, true);
+  });
+}''';
+
+    setUp(() async {
+      await d.file('weak.dart', '''
```

Comment:
[optional] we're changing terminology to "sound null safety" and "no sound null safety". Consider using `unsound.dart` here. It's not likely to confuse us, but there also isn't any reason not to go for consistency when possible.

Reply:
Changed it to opted_out.dart, and the test names are changed to use this terminology as well.